src/providers.ts

```diff
@@ -21,7 +21,13 @@ const fetchGitHubFocusViewData = async (token: string) => {
 		username: providerUser.username,
 	});
 
-	return { providerUser: providerUser, pullRequests: pullRequests };
+	return {
+		providerUser: providerUser,
+		pullRequests: pullRequests.map(pr => ({
+			...pr,
+			uniqueId: JSON.stringify(['github', 'pr', '1', '', pr.graphQLId || pr.id]),
```

Comment:
Note: the latest version of the `@gitkraken/provider-apis` package has a function for this, but the latest version also has a non-trivial breaking change to the `groupPullRequestsIntoBuckets` function, so for now I'm just reimplementing the unique id function and created a card to update the package later to prioritize bug fixes / polish.